Updating docs for sec issues (#338)
Aiven-Open/klaw-docs: docs/authentication-authorization/authentication/userpwd-db.md

    `klaw.jasypt.encryptor.secretkey=kw2021secretkey`

    Additionally, bcrypt encryption is supported for enhanced security starting with Klaw 2.10.0.

> 

Reviewer: I think that we need to put a note here, that if users update their password after 2.10.0 and they decide to revert to <2.10.0 they will need to reset their passwods.

>     Additionally, bcrypt encryption is supported for enhanced security. (starting from the Klaw 2.9.0 release)

Reviewer 2: ```suggestion
   As of Klaw 2.9.0, bcrypt encryption is supported for enhanced security.
```

is "bcrypt encryption" the correct term, or should we use "bcrypt password hashing"?
like - As of Klaw 2.9.0, bcrypt password hashing is supported for enhanced security.

Author: Indeed its password hashing. @aindriu-aiven implemented this feature. 
Let's see if we get more inputs from Aindriu too.

Reviewer: BCrypt was actually always being used, however we are now using BCrypt at rest to protect passwords.

Previously BCrypt was only being used when the passwords were added to the in Memory Database used for authentication.

The other thing is it will be from Klaw 2.10.0 as 2.9.0 is already released.

I think we can possibly re-word it to 

``Additionally, bcrypt encryption has been added for enhanced password security, where all new and or updated passwords will use this encryption going forward. (starting from the Klaw 2.10.0 release)``

Reviewer 2: Purely from an editorial pov, can you write this as: 

`Additionally, bcrypt encryption is supported for enhanced security starting with Klaw 2.9.0.`
